content/docs/quickstart/_index.md

```diff
@@ -3,7 +3,13 @@ title: Get Started
 no_on_this_page: true
 ---
 
-Get started provisioning infrastructure with Pulumi.
+<div class="flex max-w-2xl">
+
+    <p class="text-xl leading-loose text-gray-600">
+        Step-by-step guides for creating, deploying, and managing infrastructure with
+        Pulumi on the cloud using your favorite language
```

Comment:
This is probably squishy grammatical territory since it's an intro, but it feels like a period would be appropriate here. I see there was one there before, so maybe keep that?

Reply:
I've addressed all feedback except for this one: Definitely squishy territory but we can leave the period out since we've changed the heading from being a directive to a descriptive phrase/tag line. :)